app/views/IndexView.scala.html

```diff
@@ -193,14 +200,17 @@ <h2 class="govuk-heading-l">@messages("account.homePage.title")</h2>
             links = Seq[Option[Html]](
                 returnsLink,
                 Some(link(id = "guidance-link", text = messages("account.homePage.card.makeReturn.guidance.link"), call = Call("GET", completeReturnUrl), newTab = true))
-            ).filter(_.isDefined).flatten,
+            ).filter(_.isDefined).flatten
         )
 
+        @* <!-- todo should the link always show? --> *@
```

Comment:
think the answer to this is yes, there is no issue if the user follows this link even if amount is 0 more of a UCD question.

Reply:
I agree - allows user to still make a payment even if we're out of date and think they currently owe nothing